Components/WhyChooseUs.tsx

```diff
@@ -33,11 +33,12 @@ const WhyChooseUs = () => {
 
 const InfoCard = ({ title, description }: any) => {
   return (
-    <div className=" p-4 hover:bg-seafoam transition-colors font-archivo rounded-3xl ">
-      {/* <span className=" absolute inline-flex -z-10 mt-2 h-4 w-20 rounded-full bg-seafoam opacity-75"></span> */}
+    <div className=" p-4 hover:bg-indigo-950 transition-colors font-archivo rounded-3xl ">
```

Comment:
instead of using a blue shade `hover:bg-indigo-950` please use green shade, so that I can match with the image.

Reply:
Like light green , or dark green or which?